client/landing/stepper/declarative-flow/internals/steps-repository/sensei-plan/sensei-plan-products.ts

```diff
@@ -1,12 +1,9 @@
+import { ProductsList } from '@automattic/data-stores';
 import { useLocale } from '@automattic/i18n-utils';
 import { useSelect } from '@wordpress/data';
 import { useSupportedPlans } from 'calypso/../packages/plans-grid/src/hooks';
-import { PLANS_STORE, PRODUCTS_LIST_STORE } from 'calypso/landing/stepper/stores';
-import type {
-	PlanBillingPeriod,
-	PlansSelect,
-	ProductsListSelect,
-} from 'calypso/../packages/data-stores';
+import { PLANS_STORE } from 'calypso/landing/stepper/stores';
+import type { PlanBillingPeriod, PlansSelect } from 'calypso/../packages/data-stores';
```

Comment:
While we're here, could we change this to import from `@automattic/data-stores`?

Reply:
Good call - done in 765618900a235619bc9f7969ac1859c334fd50f3.